lib/task_helper.rb

```diff
@@ -33,7 +33,7 @@ def platform_is_windows?(platform)
   # (plain_windows)            webserserver-windows-2008
   # (bare_win_with_demlimiter) webserver-win-2008
   windows_regex = %r{(?<plain_windows>windows)|(?<bare_win_with_demlimiter>(?:^|[\/:\-\\;])win(?:[\/:\-\\;]|$))}
```

Comment:
You can also just add the `i` flag at the end

` windows_regex = %r{(?<plain_windows>windows)|(?<bare_win_with_demlimiter>(?:^|[\/:\-\\;])win(?:[\/:\-\\;]|$))}i`

Reply:
``` text
windows_regex = %r{(?<plain_windows>windows)|(?<bare_win_with_demlimiter>(?:^|[\/:\-\\;])win(?:[\/:\-\\;]|$))}i|(?<bare_win_with_demlimiter>(?:^|[\/:\-\\;])win(?:[\/:\-\\;]|$))}i
=> /(?<plain_windows>windows)|(?<bare_win_with_demlimiter>(?:^|[\/:\-\\;])win(?:[\/:\-\\;]|$))/i
[12] pry(main)> "windows" =~ windows_regex
=> 0
[13] pry(main)> "Windows" =~ windows_regex
=> 0
[14] pry(main)>
```

Reply:
Also...this is just _screaming_ for a test here.... 😁

Reply:
Testing?

![gif of man on fire](https://media.giphy.com/media/xXvIkTu08XQLC/giphy.gif)